errata_tool/erratum.py

```diff
@@ -573,15 +573,18 @@ def addBuildsDirect(self, buildlist, release, **kwargs):
     def addBuilds(self, buildlist, **kwargs):
         if self._new:
             raise ErrataException('Cannot add builds to unfiled erratum')
-        if 'release' not in kwargs:
```

Comment:
This looks sane, but we may want to consider restacking these to check for 'release' in kwargs, then self._release and finally single build same release case.  Otherwise raise exception that Need to specify a release

Reply:
was thinking about something like this as a followup to simplify the logic https://github.com/yazug/errata-tool/commit/18faed1e4b7bd21cc55063e5b03aea81e2bba214